docs/development/core/server/kibana-plugin-core-server.opsmetrics.md

```diff
@@ -19,7 +19,7 @@ export interface OpsMetrics
 |  [collected\_at](./kibana-plugin-core-server.opsmetrics.collected_at.md) | <code>Date</code> | Time metrics were recorded at. |
 |  [concurrent\_connections](./kibana-plugin-core-server.opsmetrics.concurrent_connections.md) | <code>OpsServerMetrics['concurrent_connections']</code> | number of current concurrent connections to the server |
 |  [os](./kibana-plugin-core-server.opsmetrics.os.md) | <code>OpsOsMetrics</code> | OS related metrics |
-|  [process](./kibana-plugin-core-server.opsmetrics.process.md) | <code>OpsProcessMetrics</code> | Process related metrics. Deprecated in favor of processes field. |
+|  [process](./kibana-plugin-core-server.opsmetrics.process.md) | <code>OpsProcessMetrics</code> | Process related metrics. |
```

Comment:
@Bamieh do you know why this is being updated here? I see you just recently added this.

Reply:
cc @elastic/kibana-core

Reply:
Our doc system doesn't re-generate documentation if an interface is not changed. So a comment change won't trigger `*.md` file re-build. That's likely why this change didn't appear in Ahmad's PR.
Unfortunately, Github UI dies on the diff, so it's hard to say *why* it appeared in your PR. Maybe you ran the doc gerneation script with the `--docs` flag or you changed an interface?

Reply:
That makes sense, thanks. I am also seeing this change when building on `master`.